### PR TITLE
Issue #13 - Update Readme

### PR DESCRIPTION
**Pull request by Bagurp**
* Updates README.md
* Adds CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Pull requests require a minimum of one reviewer to accept changes prior to merge. It is the responsibility of the person submitting the pull request to schedule a code review.
+
+[Google Engineering - Code Review Developer Guide](https://google.github.io/eng-practices/review/)
+
+## Deploy
+<a id="markdown-deploy" name="deploy"></a>
+*Deployment steps from the perspective of an engineer required to deploy to all of the available evironments.*
+
+Once a pull request is merged the changed will be automatically deployed to the appropriate environment.
```
Comment (hwilliams314) [inline CONTRIBUTING.md:95]: changed here should be just change

Comment (Bagurp) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+<a id="markdown-contributing" name="contributing"></a>
+*Detailed expectations for those contributing to this repository. A documented branching strategy is recommended. Trunk based development as default.*
+
+This project uses a [trunk based development](https://trunkbaseddevelopment.com/) branching strategy.
+
+[Trunk-based Development vs. Git Flow](https://www.toptal.com/software/trunk-based-development-git-flow)
+
+## Branches
+<a id="markdown-branches" name="branches"></a>
+*Branch naming must be consistent. Additonal patterns above case may be added as necessary. Automated enforcement in CI is recommended.*
```
[the remark this answers is not in the document]
Comment (bhatiam12) [reply CONTRIBUTING.md:27]: If everyone agrees then I think adding story# to the branch name works well. for example: feature/S13-update-readme. 
Also, we may want to add "release/*" for release branches.

[the remark this answers is not in the document]
Comment (mkimberlin) [reply CONTRIBUTING.md:27]: Not sure we will have the need for release branches for a while, but I'm ok with adding it.

I'm down with adding issue number as well, but I tend to prefer having it included in the prefix: `feature-13/update-readme`.  If the rest of the team prefers something else though, I really don't have a strong opinion on it. It's just my familiarity bias. ;-)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+<!-- /TOC -->
+
+# Contributing
+<a id="markdown-contributing" name="contributing"></a>
+*Detailed expectations for those contributing to this repository. A documented branching strategy is recommended. Trunk based development as default.*
+
+This project uses a [trunk based development](https://trunkbaseddevelopment.com/) branching strategy.
+
+[Trunk-based Development vs. Git Flow](https://www.toptal.com/software/trunk-based-development-git-flow)
```
[the remark this answers is not in the document]
Comment (Bagurp) [reply CONTRIBUTING.md:23]: I agree, I think git-flow development model suits this project. I could use this wiki in the documentation - https://nvie.com/posts/a-successful-git-branching-model/

[the remark this answers is not in the document]
Comment (jorwalk) [reply CONTRIBUTING.md:23]: I think we can all agree `git-flow` is the preferred branching strategy.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jorwalk) [reply CONTRIBUTING.md:27]: I am okay either way. At the end of the day they need to be able to be merged.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+<a id="markdown-deploy" name="deploy"></a>
+*Deployment steps from the perspective of an engineer required to deploy to all of the available evironments.*
+
+Once a pull request is merged the change will be automatically deployed to the appropriate environment.
+
+# Task Workflow
+<a id="markdown-task-workflow" name="task-workflow"></a>
+*Description of how tasks are tracked through project management software and expectations of different roles.*
+
+![Task Workflow Diagram](task%20workflow.png)
```
[the remark this answers is not in the document]
Comment (jorwalk) [reply CONTRIBUTING.md:101]: @Bagurp has this been updated or resolved?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+## Branches
+<a id="markdown-branches" name="branches"></a>
+*Branch naming must be consistent. Additonal patterns above case may be added as necessary. Automated enforcement in CI is recommended.*
+
+Branch names must be in [kebab-case](https://en.toolpage.org/tool/kebabcase).
+
+## Commits
+<a id="markdown-commits" name="commits"></a>
+*Consistent, verbose commits reduce the need for additional documentation. May be used for generating changelogs. Detailed instructions must be included either through a documented specification or custom documentation. Automated enforcement in CI is recommended.*
+
```
[the remark this answers is not in the document]
Comment (jorwalk) [reply CONTRIBUTING.md:34]: @Bagurp can you remove the last sentence and them @mkimberlin can resolve.

**Comment by jorwalk** [comment]
@mkimberlin once the changes to the Contributing documentation are made, could you please remove the `request change`

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Bagurp) [reply CONTRIBUTING.md:23]: Updated text. Should be resolved with the new changes.

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bagurp) [reply CONTRIBUTING.md]: Updated text. Should be resolved with the new changes.

File: CONTRIBUTING.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Bagurp) [reply CONTRIBUTING.md:42]: Updated text. Should be resolved with the new changes.

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Bagurp) [reply CONTRIBUTING.md:109]: Added the image to repo. Should be resolved with the new changes.

**Pull request merged**